minimal/app.json

```diff
@@ -0,0 +1,27 @@
+{
+  "expo": {
+    "name": "app",
+    "description": "This project is really great.",
+    "slug": "app",
+    "privacy": "public",
+    "sdkVersion": "26.0.0",
```

Comment:
Let's go ahead and jump to 27.0.0 (announced [here](https://blog.expo.io/expo-sdk-v27-0-0-is-now-available-898bf1e5b0e4)).